src/page/summary-step.page.ts

```diff
@@ -4,7 +4,7 @@ export class SummaryStepPage {
   private nextStep: ElementFinder;
 
   constructor () {
-    this.nextStep = $('.cart_navigation span');
+    this.nextStep = $('.cart_navigation > a.button');
```

Comment:
Why is this one a better selector?

Reply:
I didn't see it as a better selector. At first I tried to homogenize all the buttons of "Proceed to checkout" with the same selector, so I changed it, but as most buttons have names and it is a better solution as a selector, the change is no longer so necessary.